src/components/tabs/__internal__/tab-title/tab-title.style.ts

```diff
@@ -328,7 +333,7 @@ const tabTitleStyles = css<TabTitleProps>`
       ${borders &&
       !(noRightBorder || noLeftBorder) &&
       css`
-        &:nth-of-type(n + 1) {
+        &:nth-of-type(n + 1):not(:first-of-type) {
```

Comment:
**question**: I'm not quite sure what's going on with this change, but both here and on line 348 this presumably will affect old focus styles as well as new ones? Are we sure this is what we want?

Reply:
There is an additional border being added in both old and new world

Reply:
OK, thanks - but if it applies in the "old world" too it feels like a bit of scope creep or at least something separate which should be in a different commit?